pincer/commands/commands.py

```diff
@@ -746,7 +899,7 @@ def should_be_removed(target: AppCommand) -> bool:
             return True
 
         # NOTE: Cannot be generator since it can't be consumed due to lines 743-745
-        to_remove = list(filter(should_be_removed, self._api_commands))
+        to_remove = [*filter(should_be_removed, self._api_commands)]
```

Comment:
is star operator faster/better than `list`?

Reply:
> is star operator faster/better than `list`?

I found it about 1-2% faster so probably no difference. Looks better though

Reply:
I think the list operator is much more readable in this case